db/migrate/20230330133724_move_growth_funding_from_funding_values_and_migrate_data.rb

```diff
@@ -0,0 +1,11 @@
+class MoveGrowthFundingFromFundingValuesAndMigrateData < ActiveRecord::Migration[6.1]
+  def up
+    PafsCore::DataMigration::MoveGrowthFundingToOtherAdditionalGia.up
+    remove_column :pafs_core_funding_values, :growth_funding
```

Comment:
Do we need error handling here in case `MoveGrowthFundingToOtherAdditionalGia` fails? We don't want to remove the column (and the existing data) unless that succeeds.

Reply:
Since migrations are run inside a transaction, shouldn't it rollback if there is an error in some part anyway?

Reply:
I think so, just a bit nervous when business data is about to be deleted. Could you run a quick check, throwing an error in the first migration please?

Reply:
Yep